skactiveml/visualization/_data_sets.py

```diff
@@ -0,0 +1,116 @@
+import numpy as np
```

Comment:
We need discuss whether this file is necessary. `make_blobs` can also be used to generate such data sets.